charts/everest/templates/monitoring-resources.yaml

```diff
@@ -0,0 +1,110 @@
+{{- if .Values.createMonitoringResources }}
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMServiceScrape
+metadata:
+  name: kube-state-metrics
+  namespace: {{ .Values.monitoring.namespaceOverride }}
+  labels:
+    app.kubernetes.io/managed-by: everest
+    everest.percona.com/type: monitoring
+spec:
+  jobLabel: app.kubernetes.io/name
+  namespaceSelector: {}
+  endpoints:
+    - scheme: http
+      port: http
+      honorLabels: true
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: kube-state-metrics
+---
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMNodeScrape
+metadata:
+  name: pmm-vm-cadvisor-metrics
+  namespace: {{ .Values.monitoring.namespaceOverride }}
+  labels:
+    app.kubernetes.io/managed-by: everest
+    everest.percona.com/type: monitoring
+spec:
+  scheme: "https"
+  interval: "10s"
+  honorLabels: true
+  scrapeTimeout: "2s"
+  selector: {}
+  tlsConfig:
+    insecureSkipVerify: true
+    caFile: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+  bearerTokenFile: "/var/run/secrets/kubernetes.io/serviceaccount/token"
+  relabelConfigs:
+    - action: labelmap
+      regex: __meta_kubernetes_node_label_(.+)
+    - targetLabel: __address__
+      replacement: kubernetes.default.svc:443
+    - sourceLabels: [__meta_kubernetes_node_name]
+      regex: (.+)
+      targetLabel: __metrics_path__
+      replacement: /api/v1/nodes/$1/proxy/metrics/cadvisor
+  metricRelabelConfigs:
+  - action: labeldrop
+    regex: cloud_google_com.*
+  - source_labels:
+    - namespace
+    - pod
+    target_label: node_name
+    regex: (.+);(.+)
+    replacement: $1-$2
+---
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMNodeScrape
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: everest
+    everest.percona.com/type: monitoring
+  name: pmm-vm-kubelet-metrics
+  namespace: {{ .Values.monitoring.namespaceOverride }}
+spec:
+  bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+  honorLabels: true
+  interval: 30s
+  metricRelabelConfigs:
+  - action: labeldrop
+    regex: (uid)
+  - action: labeldrop
+    regex: (id|name)
+  - action: drop
+    regex: (rest_client_request_duration_seconds_bucket|rest_client_request_duration_seconds_sum|rest_client_request_duration_seconds_count)
+    source_labels:
+    - __name__
+  - action: labeldrop
+    regex: cloud_google_com.*
+  relabelConfigs:
+  - action: labelmap
+    regex: __meta_kubernetes_node_label_(.+)
+  - sourceLabels:
+    - __metrics_path__
+    targetLabel: metrics_path
+  - replacement: kubelet
+    targetLabel: job
+  scheme: https
+  scrapeTimeout: 5s
+  tlsConfig:
+    caFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    insecureSkipVerify: true
+---
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMPodScrape
```

Comment:
Were you able to confirm that we need this after all?

Reply:
Not yet, but once I have confirmation, I will remove it in a follow-up PR.

For now I'm just moving everything from the everest repo, so that there's track of what we moved/removed and why we did so